### PR TITLE
Adding Short Description Field in Projects

### LINKED ISSUE #109: Add tweet length description field
Hi all! We'll need to add a **"Short description"** field to the project page. The character limit should be 129 chars. (😬 cc: @caseypugh @jamiew )

This is for our new project card design - easier for users to read on home page! 

Screen Shot 2020-04-01 at 9 53 52 PM

### PR DESCRIPTION
**Pull request by bansalnagesh**
Fixes #109

### CODE REVIEW

**Comment by bansalnagesh** [comment]
image

image

I can fix copy changes if any.

File: app/views/projects/show.html.erb [hunk 1]
```diff
@@ -250,6 +250,14 @@
     <div class="px-4 py-5 sm:p-0">
       <dl>
         <div class="sm:grid sm:grid-cols-3 sm:gap-4 sm:px-6 sm:pt-5 sm:pb-2">
+          <dt class="text-sm leading-5 font-medium text-gray-500">
+            Short Description
```
Comment (sradu) [inline app/views/projects/show.html.erb:254]: Short description

File: app/views/projects/_project_form.html.erb [hunk 2]
```diff
@@ -10,6 +10,18 @@
...
 
+  <div class="mt-6 sm:mt-5 sm:grid sm:grid-cols-3 sm:gap-4 sm:items-start sm:border-t sm:border-gray-200 sm:pt-5">
+    <label for="short_description" class="block text-sm font-medium leading-5 text-gray-700 sm:mt-px sm:pt-2">
+      Describe your project in a Tweet
+    </label>
+    <div class="mt-1 sm:mt-0 sm:col-span-2">
+      <div class="max-w-lg flex rounded-md shadow-sm">
+        <%= f.text_field "short_description", maxlength: 129, class: "form-textarea block w-full transition duration-150 ease-in-out sm:text-sm sm:leading-5", required: true %>
+      </div>
+      <p class="mt-2 text-sm text-gray-500">A brief description about your project (Max 129 characters).</p>
```
Comment (sradu) [inline app/views/projects/_project_form.html.erb:21]: (max 129 characters)

File: app/views/projects/_project_form.html.erb [hunk 3]
```diff
@@ -10,6 +10,18 @@
     </div>
   </div>
 
+  <div class="mt-6 sm:mt-5 sm:grid sm:grid-cols-3 sm:gap-4 sm:items-start sm:border-t sm:border-gray-200 sm:pt-5">
+    <label for="short_description" class="block text-sm font-medium leading-5 text-gray-700 sm:mt-px sm:pt-2">
+      Describe your project in a Tweet
```
Comment (sradu) [inline app/views/projects/_project_form.html.erb:15]: in a tweet

**Pull request merged**